src/constants.js

```diff
@@ -327,3 +327,9 @@ export const SORT_DIRECTIONS = {
 export const CONSORTIA_API = 'consortia';
 export const CONSORTIA_TENANTS_API = 'tenants';
 export const CONSORTIA_USER_TENANTS_API = 'user-tenants';
+
+export const AFFILIATION_ERROR_CODES = {
```

Comment:
do you really need to store these constants on module level? I believe it's UserAffiliations level